vscode-extension/media/CreateAIConfig.md

```diff
@@ -1,5 +1,11 @@
-## Create New AIConfig
+## Create AIConfig
 
-Open the Command Palette (CMD+Shift+P) and run the command `AIConfig: Create New`
+> _Tip_: You can use the Command Palette (`CMD`+`Shift`+`P`) to run `AIConfig: Create New` to create a new aiconfig anytime.
+
+You can create the aiconfig in JSON or YAML format. The file defines the models, prompts and settings
```

Comment:
would like to say "The file defines the models, prompts and settings of your AI workflow" at the end to clarify but not a big deal